src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,20 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentrosDistribucion(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+    fun agregarVendedor(unVendedor: Vendedor) {
+        if (vendedores.contains(unVendedor)) {
+            throw Exception("el vendedor ya esta registrado")
+        } else {
+            vendedores.add(unVendedor)
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if (vendedores.contains(unVendedor)) {
            throw Exception("el vendedor ya esta registrado")
        }
        
        vendedores.add(unVendedor)
```